tests/generic_config_updater/gu_utils.py

```diff
@@ -0,0 +1,48 @@
+import json
+import time
```

Comment:
lgtm checker found it not used, and it's a true warning. Please respect lgtm alerts.